LoopOnboardingKitUI/View Controllers/OnboardingUICoordinator.swift

```diff
@@ -242,34 +242,39 @@ class OnboardingUICoordinator: UINavigationController, CGMManagerOnboarding, Pum
     }
 
     private func constructTherapySettingsViewModel(therapySettings: TherapySettings) -> TherapySettingsViewModel? {
+        return TherapySettingsViewModel(therapySettings: therapySettings, delegate: self)
+    }
+}
+
+extension OnboardingUICoordinator: TherapySettingsViewModelDelegate {
+    func syncBasalRateSchedule(items: [RepeatingScheduleValue<Double>], completion: @escaping (Result<BasalRateSchedule, Error>) -> Void) {
+        // noop
+    }
+    
+    func syncDeliveryLimits(deliveryLimits: DeliveryLimits, completion: @escaping (Result<DeliveryLimits, Error>) -> Void) {
+        // noop
+    }
+    
+    func saveCompletion(for therapySetting: TherapySetting, therapySettings: TherapySettings) {
+        stepFinished()
+    }
+    
+    func pumpSupportedIncrements() -> PumpSupportedIncrements? {
         let supportedBasalRates = (1...600).map { round(Double($0) / Double(1.0/0.05) * 100.0) / 100.0 }
 
         let maximumBasalScheduleEntryCount = 24
 
         let supportedBolusVolumes = (1...600).map { Double($0) / Double(1/0.05) }
-
-        let pumpSupportedIncrements = PumpSupportedIncrements(
+        
+        return PumpSupportedIncrements(
             basalRates: supportedBasalRates,
             bolusVolumes: supportedBolusVolumes,
             maximumBasalScheduleEntryCount: maximumBasalScheduleEntryCount
         )
-
-        return TherapySettingsViewModel(
-            therapySettings: therapySettings,
-            pumpSupportedIncrements: { pumpSupportedIncrements },
-            syncPumpSchedule: {
-                { _, _ in
-                    // Since pump isn't set up, this syncing shouldn't do anything
```

Comment:
[nit] If you don't go the NOP route above, consider adding this comment and assertion to both `syncBasalRateSchedule` and `syncDeliveryLimits` above.